### PR TITLE
1 Lab 🥲

### PR DESCRIPTION
**Pull request by justmark0**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+repos:
```
Comment (DOCtorActoAntohich) [inline .pre-commit-config.yaml:1]: dude i love/hate pre-commit

**Review by DOCtorActoAntohich — APPROVED** [review]
gud :trollface:

**Comment by Andrew174194** [comment]
Good job, the only problem here I see that you made a PR to master branch, not lab1 as it should be...🤔

File: app_python/app.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import pytz
+import requests
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def moscow_time():
+    url = "http://worldtimeapi.org/api/timezone/Europe/Moscow"
```
Comment (ntdesmond) [inline app_python/app.py]: From <http://worldtimeapi.org/pages/faqs#please-do-not>:
>Please note that there is a fair-use policy in place; any attempt to make many requests in a short timeframe will find the requests are forbidden.
>
>Therefore, this API should not be used:
>
> - in "real-time" situations where absolutely accurate timing data is required
> - as an alternative to [ntp](https://en.wikipedia.org/wiki/Network_Time_Protocol)
> - as an alternative to calculating local time changes (eg. making a request every second)

While we were not told that we're going to refresh the page every second, this still might be an issue in some cases. 

It's better not to call an external API on each incoming request. Besides the reason above, you also can't guarantee that the API you call will respond you in a short timespan (let's say, less that a second). A few seconds of delay and you'll tell the user the wrong time (with a delay, too).

**Review by ntdesmond — APPROVED** [review]
🐍

**Comment by justmark0** [comment]
> Good job, the only problem here I see that you made a PR to master branch, not lab1 as it should be...🤔

Yeah, here is link for new pr https://github.com/innopolis-devops/labs/pull/79 .  I will close this pr and submit this link because TA should see that this pr were reviewed

**Pull request closed without merging**